libnavigation-core/src/test/java/com/mapbox/navigation/core/internal/trip/service/MapboxTripServiceTest.kt

```diff
@@ -112,4 +126,38 @@ class MapboxTripServiceTest {
 
         verify(exactly = 1) { tripNotification.updateNotification(routeProgress) }
     }
+
+    @Test
+    fun notificationDataPostToChannelOnServiceStarted() {
+        runBlocking {
+            service.startService()
+
+            val notificationData = MapboxTripService.getNotificationDataChannel().receive()
+
+            Assert.assertEquals(NOTIFICATION_ID, notificationData.notificationId)
```

Comment:
```suggestion
           assertEquals(NOTIFICATION_ID, notificationData.notificationId)
```

Same applies to the rest of the tests 👇

Reply:
fixed